app/views/newfromtemplate.html

```diff
@@ -16,6 +16,7 @@
                   {{ emptyMessage }}
                 </div>
                 <div class="osc-form" ng-show="template">
+                  <alerts alerts="alertsTop" hide-close-button="true"></alerts>
```

Comment:
yeah they were moved for the quota change because you would submit the form and not see the errors if they were scrolled off the top of the page